src/AsyncSessionMiddlewareFactory.php

```diff
@@ -2,32 +2,39 @@
 
 namespace Interop\Session\Middleware\Async;
 
-use Psr\Http\Message\{ServerRequestInterface as Request, ResponseInterface as Response};
-use Interop\Session\Manager\{SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager};
+use Psr\Http\Message\{
+    ServerRequestInterface as Request, ResponseInterface as Response
+};
+use Interop\Session\Manager\{
+    SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager
+};
 use Interop\Session\Configuration\SessionConfigurationInterface;
 
 use Interop\Session\SessionInterface;
 
-class AsyncSessionMiddlewareFactory {
+class AsyncSessionMiddlewareFactory
```

Comment:
But do you have an actual use case?
The container must both be container-interop compatible and not service-provider compatible. There are not that many out there...
If this code is not actually used, I'd rather remove it (because if it ain't tested, it ain't working)